### DISCUSSION TITLE
rtorrent help

### DISCUSSION THREAD

**Discussion by MQB23** [rakshasa/rtorrent#1198] [General]
Hi, i have just make a mistake and open rtorrent, when one of my partition wasnt mounted at all. So i just lost torrent state for torrents that where on this partition... all of this torrents are in CLOSED state and need to be rehash again.... is there any way to recover from this disaster? Its a lot of data...

**Comment by kannibalox** [comment]
If you know they are all 100% complete, you can play some tricks with fast_resume, but if you don't know what state they're in it's better to just accept the rehash. In the future, you may want to consider backing up your rtorrent session directory, in case of this or any other problems.

**Comment by MQB23** [comment]
they are 100% complete... can you give example with fast_resume?
